src/honey-garlic-chicken.md

```diff
@@ -0,0 +1,33 @@
+# Honey Garlic Chicken with Broccoli and Brown Rice
+
+Very easy, high protein.
+
+- Servings: 2
+
+## Ingredients
+
+- 1/2 cup brown rice
+- 1/2 cup frozen peas
```

Comment:
```suggestion
- 1/2 cup ( 70g ) frozen peas
```

Reply:
Keep the parenthesis formatting consistent. Should be `(70g)`.

Reply:
My mistake, above is now ( 100g ). Even tho you forgot to use the magic word @DGKSK8LIFE xD